mcc/odin/odin/snapshot_list_test.go

```diff
@@ -2,25 +2,41 @@ package odin_test
 
 import (
 	"testing"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/rds"
 
 	"github.com/poka-yoke/spaceflight/mcc/odin/odin"
 )
 
+const (
+	RFC8601 = "2016-01-02T15:04:05+07:00"
```

Comment:
@dsager Exactly that. Apply your format to a specific agreed upon point in time and reverse the parsing from it programatically. It's supposed to avoid many subtle and common bugs.